modules/flowable-rest/src/main/java/org/flowable/rest/service/api/runtime/process/ProcessInstanceIdentityLinkResource.java

```diff
@@ -72,9 +78,15 @@ public void deleteIdentityLink(@ApiParam(name = "processInstanceId") @PathVariab
             @ApiParam(name = "type") @PathVariable("type") String type,
             HttpServletResponse response) {
 
-        ProcessInstance processInstance = getProcessInstanceFromRequest(processInstanceId);
+        ProcessInstance processInstance = runtimeService.createProcessInstanceQuery().processInstanceId(processInstanceId).singleResult();
+        if (processInstance == null) {
+            throw new FlowableObjectNotFoundException("Could not find a process instance with id '" + processInstanceId + "'.");
+        }
 
         validateIdentityLinkArguments(identityId, type);
+        if (restApiInterceptor != null) {
+            restApiInterceptor.deleteProcessInstanceIdentityLink(processInstance, identityId, type);
```

Comment:
Same story as for deleting a case instance identity link

Reply:
done